dace/runtime/include/dace/xilinx/stream.h

```diff
@@ -14,13 +14,11 @@ namespace dace {
 template <typename T, unsigned vector_length, unsigned capacity>
 class FIFO {
  public:
-  FIFO() : stream_(capacity) {
+  FIFO() : stream_() {
     #pragma HLS INLINE
-    #pragma HLS STREAM variable=stream_ depth=capacity
   }
-  FIFO(char const *const name) : stream_(name, capacity) {
+  FIFO(char const *const name) : stream_(name) {
     #pragma HLS INLINE
-    #pragma HLS STREAM variable=stream_ depth=capacity
```

Comment:
What's happening here? Are streams unbounded now?

Reply:
Xilinx changed how you specify the stream depth, it's still being passed as a template parameter (see bottom of class).